service/build.gradle

```
@@ -112,3 +116,25 @@ tasks.register("pactTests", Test) {
 		showStandardStreams = true
 	}
 }
+
+testing {
+    suites {
+        integrationTest(JvmTestSuite) {
+            dependencies {
+        		implementation project()
+        		implementation project(':client')
```

I think you can replace these two lines with:
```suggestion
        		implementation project(':service')
```